astroquery/query.py

```diff
@@ -4,12 +4,13 @@
 import abc
 import pickle
 import hashlib
+import io
```

Comment:
Note: Technically, this can be imported inside the `else` block. Readability vs import time. Let me know of your preference.

Reply:
readibility is fine I feel, import time is never really a bottleneck when your package downloads fits files from the internet ;)